test/fixtures/context.ts

```diff
@@ -127,7 +127,7 @@ export async function contextFixture(
 
   if (users) {
     users.forEach((user) => {
-      data.users.create(user.tenant_id, user);
+      data.users.create("tenantId", user);
```

Comment:
but why? I'd hope a test would fail here and that we're testing more than one tenant... Maybe these aren't used on the hono/router tests